WooCommerce/Classes/Extensions/UIView+SafeAreaConstraints.swift

```diff
@@ -0,0 +1,58 @@
+import UIKit
+
+extension UIView {
+    var safeLeadingAnchor: NSLayoutAnchor<NSLayoutXAxisAnchor> {
+        get {
+            if #available(iOS 11.0, *) {
+                return self.safeAreaLayoutGuide.leadingAnchor
```

Comment:
nit: `self` might not be needed? (and same for the other `self` usage in this file)